### ISSUE TITLE
New File

### ISSUE THREAD

**Issue by wbrawner** [wbrawner/SimpleMarkdown#2]
Add menu item to create new file (prompt to save current data and then clear it)

**Comment by lokesh-krishna** [comment]
Sorry about bumping this but I'm loving this app so much and not being able to create a new file without first swiping it off my list of recent apps really gets in the way of using it.  

Is there any other workaround until this gets implemented?

**Comment by wbrawner** [comment]
I agree, it's a pain. For now what I've been doing is long tapping the editable area, tapping "Select all", deleting the text, and saving under a different name. While not ideal, I do hope to get this feature implemented shortly :)

**Comment by wbrawner** [comment]
Implemented with #18 so it'll be out in the next release!
